process-invoices/empty.xml

```diff
@@ -1,6 +1,6 @@
 <invoice>
  <payment>
-   <voucher_date>2021-09-03Z</voucher_date>
+   <voucher_date>2021-07-22T12:00:00Z</voucher_date>
```

Comment:
@gam5 or @adamshire123 just want to double check that the time here should be `12:00:00` and not `00:00:00`, is that correct?

Reply:
yes, 12:00:00 because we want it squarely in the middle of the day to ensure it happens for the day after timezone corrections was how it was explained to me.

Reply:
Great, thanks @zotoMIT! Just wanted to make sure my perl script replacement does the right thing.